ntia_conformance_checker/cli_utils.py

```diff
@@ -0,0 +1,241 @@
+    "quiet": "No output unless there are errors",
+}
+
+
+def do_parsed_args() -> argparse.Namespace:
+    """Parse command line arguments."""
+
+    epilog_text = (
+        "choices:\n"
+        "  SBOM specifications (for --sbom-spec):\n"
+        + "\n".join(
+            f"    {k:<11} {v}" for k, v in sorted(SUPPORTED_SBOM_SPECS_DESC.items())
+        )
+        + "\n\n"
+        "  Compliance standards (for --comply):\n"
+        + "\n".join(
+            f"    {k:<11} {v}"
+            for k, v in sorted(SUPPORTED_COMPLIANCE_STANDARDS_DESC.items())
+        )
+        + "\n\n"
+        "  Output types (for --output):\n"
+        + "\n".join(f"    {k:<11} {v}" for k, v in sorted(_OUTPUT_CHOICES.items()))
+        + "\n\n"
+        "Examples:\n"
+        "  sbomcheck sbom.spdx\n"
+        "  sbomcheck --output json --output-file report.json sbom.yaml\n"
+        "  sbomcheck --sbom-spec spdx3 --skip-validation sbom.json\n"
+    )
+
+    parser = argparse.ArgumentParser(
+        description="Check if an SPDX SBOM complies with a compliance standard.",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog=epilog_text,
+    )
+    parser.add_argument(
+        "file", metavar="PATH", nargs="?", help="Filepath for SBOM input"
+    )
+    parser.add_argument(
+        "--file",  # for backward compatibility
+        dest="file_opt",
+        help=argparse.SUPPRESS,  # hide from help
+    )
+    parser.add_argument(
+        "-s",
+        "--sbom-spec",
+        choices=sorted(SUPPORTED_SBOM_SPECS),
+        default=DEFAULT_SBOM_SPEC,
+        help=(
+            "SBOM specification of the input file; see below for details "
+            f"[default: {DEFAULT_SBOM_SPEC}]"
+        ),
+    )
+    parser.add_argument(
+        "-c",
+        "--comply",
+        choices=sorted(SUPPORTED_COMPLIANCE_STANDARDS),
+        default=DEFAULT_COMPLIANCE_STANDARD,
+        help=(
+            "Compliance standards to check against; see below for details "
+            f"[default: {DEFAULT_COMPLIANCE_STANDARD}]"
+        ),
+    )
+    parser.add_argument(
+        "--conform",  # alias of --comply
+        dest="comply",
+        help=argparse.SUPPRESS,  # hide from help
+    )
+    parser.add_argument(
+        "--skip-validation",
+        action="store_true",
+        default=False,
+        help="Skip validation",
+    )
+    parser.add_argument(
+        "--output",
+        choices=sorted(_OUTPUT_CHOICES),
+        default="print",
+        help="Type of compliance report output; see below for details [default: print]",
+    )
+    parser.add_argument(
+        "-o",
+        "--output-file",
+        metavar="PATH",
+        help="Filepath for compliance report output; if omitted, prints to console",
+    )
+    parser.add_argument(
+        "--output_path",  # for backward compatibility
+        dest="output_file",
+        help=argparse.SUPPRESS,  # hide from help
+    )
+    # parser.add_argument(
```

Comment:
Should this `--quiet` code simply be omitted rather than commented out? Why keep it around?

Reply:
I will remove this. Thank you